### PR TITLE
Feauture signup

### PR DESCRIPTION
**Pull request by ampaire**
This Pull request fixes bugs in the code

### CODE REVIEW

**Review by Oghenebrume50 — APPROVED** [review]
Hi team, good work, but the Html is not validated, please it is required do that before submitting this project the errors are not much should be about two detected, that's why I would let this slide, but make sure to fix them before submitting. :+1:  Good one team

**Comment by ampaire** [comment]
Thank you

File: index.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    </head>
+    <body>
+        <header>
+            <div class="nav-text">
+                <p>Already have an Intuit account?</p>
+                <a href="#">I forgot my User ID or Password</a>
+            </div>
+            <button class="nav-btn">Sign in</button>
+        </header>
+        <main>
```
Comment (samgaco) [inline index.html:20]: You forgot to close the `<main>` tag. [More info.](https://validator.w3.org)

**Review by samgaco — APPROVED** [review]
### Hi @ampaire and @Elijahscriptdev :vulcan_salute: 

### Everything looks correct so you are approved :muscle: 

Please :pray: **before merging address the errors given to you by the** [validator](https://validator.w3.org).  Apparently, you forgot to close one tag.

### Once you correct that, you can merge to the master branch and submit the completion form :muscle: Congratulations :tada:

**Pull request closed without merging**